### PR TITLE
instructions for joining testnet if block reward is split

### PR DESCRIPTION
**Pull request by sennui**

### CODE REVIEW

File: docs/configuration.md [hunk 1]
```diff
@@ -107,6 +107,18 @@ fork_management:
...
+Mainnet values are included in the base code, however for testnet use below:
+
+```yaml
+chain:
+    protocol_beneficiaries_enabled: true
+    protocol_beneficiaries: ["ak_2A3PZPfMC2X7ZVy4qGXz2xh2Lbh79Q4UvZ5fdH7QVFocEgcKzU:100"]
+```
+
+
 For mainnet network the network ID defaults to `ae_mainnet`.
```
Comment (dincho) [inline docs/configuration.md]: I think this should be kept under Network ID section ?

Comment (lucafavatella) [reply docs/configuration.md]: Well spotted!

File: docs/configuration.md [hunk 2]
```diff
@@ -107,6 +107,18 @@ fork_management:
     network_id: ae_uat
 ```
```
Comment (lucafavatella) [inline docs/configuration.md]:
```suggestion
For mainnet network the network ID defaults to `ae_mainnet`.

```

File: docs/configuration.md (hunk 1 above)
Comment (lucafavatella) [inline docs/configuration.md]:
```suggestion
```

File: docs/configuration.md [hunk 3]
```diff
@@ -106,9 +106,21 @@ The testnet (internally called UAT) has the network ID `ae_uat` - this is set in
...
 
+## Block reward initiative
+
+The new schema of managing block reward can be tuned with following configuration parameters.
+Mainnet values are included in the base code, however for testnet use below:
+
+```yaml
+chain:
+    protocol_beneficiaries_enabled: true
+    protocol_beneficiaries: ["ak_2A3PZPfMC2X7ZVy4qGXz2xh2Lbh79Q4UvZ5fdH7QVFocEgcKzU:100"]
```
Comment (lucafavatella) [inline docs/configuration.md]: Do we plan to update share with same as mainnet?

**Pull request merged**